.github/workflows/code-coverage.yml

```diff
@@ -0,0 +1,46 @@
+name: omise-go code-coverage
+
+on:
+  push:
+
+jobs:
+  sonarcloud:
+    name: SonarCloud
+    runs-on: ubuntu-latest
+    env:
+      GOPATH: ${{ github.workspace }}
+      WORKDIR: src/github.com/${{ github.repository }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0  # Shallow clones should be disabled for a better relevancy of analysis
+
+      - name: Set up Go ${{ matrix.go }}
+        uses: actions/setup-go@v2
```

Comment:
Any reason why we are not using `setup-go@v3` or `setup-go@v4`?

Reply:
Resolved